scipy/optimize/zeros.c

```diff
@@ -129,11 +129,15 @@ call_solver(solver_type solver, PyObject *self, PyObject *args)
                         "Failed to converge after %d iterations.",
                         solver_stats.iterations);
                 PyErr_SetString(PyExc_RuntimeError, msg);
-                flag = 1;
                 return NULL;
             }
+            flag = CONVERR;
         }
     }
+    else
```

Comment:
`else` and `LEFT_BRACE` on the same line would match the style used in the rest of the file.

```
else {
    flag = CONVERGED;
}
```